enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -1034,54 +1013,6 @@ _Do we need a new config object? How can the cluster admin configure OSSM?_ for
 details.). In tech preview, we may expose a customization option for enabling/disabling
 Automatic Deployments.
 
-#### Should we enable Gateway API's [ReferenceGrant](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.ReferenceGrant) CRD?
```

Comment:
Why delete this?  You could update it if needed.

Reply:
We usually keep the open question but record the answer once we have one.  This provides useful context for anyone who might have the same question or want to understand the discussion and decisions that went into the EP.

Reply:
Yes, we keep answered questions in the doc rather than removing them.

Reply:
Sounds good. I had noticed that in other sections, but kinda thought since so much of it referenced the history with OSSM 2.x which was never beyond dev preview and not relevant to our GA that it might be unnecessary. However, sounds like it's wanted so I put that back and updated the content for recent decisions, LMKWYT :vulcan_salute: